trino/types.py

```diff
@@ -101,16 +102,18 @@ def new_instance(self, value: datetime, fraction: Decimal) -> TimestampWithTimeZ
 
     def normalize(self, value: datetime) -> datetime:
         if tz.datetime_ambiguous(value):
-            return self._whole_python_temporal_value.tzinfo.normalize(value)
+            # This appears to be dead code since tzinfo doesn't actually have a `normalize` method.
+            # TODO: Fix this or remove.
```

Comment:
Please create an issue.

Reply:
I plan to send a follow-up PR for this.

Looks like it'll be a bugfix. Seems that the ambiguous date-time handling hasn't been working since the beginning.
This code became "broken" (ie. calling non-existent method) in https://github.com/trinodb/trino-python-client/commit/2b9ca0c2e59496e168907fec1df7a50f26976dce#diff-cf9b945b8cb541d38680c39caa2de6568793a6d53644d6f2659de6a77ff96127R1040

I checked both before and after that commit/PR and the code isn't called by any tests.

Will create an issue to track.

Reply:
https://github.com/trinodb/trino-python-client/issues/449